.github/workflows/buid_and_release.yml

```diff
@@ -13,15 +13,18 @@ on:
 jobs:
   build_and_release:
     runs-on: ubuntu-latest
+#    container: obolibrary/odkfull:v1.4.1
+#    container: obolibrary/odkfull:dev
+    container: obolibrary/odkfull:latest
     steps:
     - uses: actions/checkout@v2
     - name: Set up Python 3.10
       uses: actions/setup-python@v2
```

Comment:
No need to set up python, its already in ODK

Reply:
Yeah I overlooked that and found that out the hard way. It was pointed out to me as well by: https://github.com/INCATools/ontology-development-kit/issues/888